### ISSUE TITLE
Hand render for poker vlog 

### ISSUE THREAD

**Issue by lei-cao** [vikcch/riropo#1]
Hi Vikcch,

Not sure if this is a good place to leave this message.

I came across your Live Squeezer and Riropo, I found them very easy to use to record my live hands. Thank you for making them and open sourced them.
Basically, I am a poker vlogger. I record my hand's videos while playing and make youtube videos. one example here https://youtu.be/bk4HjoOXXBs?t=841
Not like other vlogger, I prefer to put the full hand actions, only important ones due to the limited spaces. May other vloger only show them like a 'title' so when I watch the videos, I kind of forgot the actions sometimes.  Currently, I use keynotes to create the hand actions manually. Then export it as a transparent background video. 
This is really time-consuming. So I want to look for a program to do it. Then I came across your awesome projects. 

I am able to run your riropo at my local machine now. So I am planning to create a render for my vlog, similar as the one I am making from keynotes. And maybe another format as other vloger are making. Then I can record it as a green screen video then edit it together with other video parts. 

So basically I am writing to you here now to ask if you are interested to build such 'render' for vloger. If not, I am wondering if you want to accept my contribution for such 'render'. If still not, I will still build it on top of your current awesome work as my own project. 

Cheers,
Lei

**Comment by vikcch** [comment]
Hi Lei,

I am happy to do it, please check your email.
